HW3/src/js/task1.js

```diff
@@ -0,0 +1,7 @@
+let a = parseFloat(prompt('Введіть суму'));
```

Comment:
please change variables name for more understanding of what they are charged of

Reply:
+